File: C/1020.c
#include <stdio.h>

int main() { 
    int n, y, y_r, m, d;

    scanf("%d", &n);

    y = n / 365;
    y_r = n % 365;

    m = y_r / 30;
    d = y_r % 30;

    printf("%d ano(s)\n", y);
    printf("%d mes(es)\n", m);
    printf("%d dia(s)\n", d);

    return 0;
}

File: Lua/1020.lua
n = tonumber(io.read())

y = math.floor(n / 365)
y_r = n % 365

m = math.floor(y_r / 30)
d = y_r % 30

print(y .. ' ano(s)')
print(m .. ' mes(es)')
print(d .. ' dia(s)')